delivery_schenker/models/schenker_request.py

```diff
@@ -72,17 +76,27 @@ def _process_reply(self, service, vals=None, send_as_kw=False):
                             "Error in the request to the Schenker API. This is the "
                             "thrown message:\n\n"
                             "[%s]\n"
-                            "%s - %s" % (error_text, error_code, error_message)
+                            "%s - %s\n"
+                            "Request Data:\n"
+                            "%s" % (error_text, error_code, error_message, data)
                         )
                     )
                 except ValidationError:
                     raise
                 # If we can't get the proper exception, fallback to the first
                 # exception error traceback
                 except Exception:
-                    raise Fault(e)
+                    msg = e.message + _("\nRequest Data:\n%s") % data
+                    raise Fault(msg)
         return response
 
+    def _dump_request_data(self, data):
+        res = json.dumps(data, indent="    ", default=str)
+        if self.access_key:
+            res = res.replace(self.access_key, "<REMOVED>")
+
+        return res
```

Comment:
I would prefer to not use str.replace, instead removing the actual keys which holding the access key.